packages/bot-web-ui/src/components/flyout-block-group.jsx

```diff
@@ -26,39 +29,42 @@ const FlyoutBlockGroup = ({ onInfoClick, block_node, is_active, should_hide_disp
     );
 
     return (
-        <div className={classNames('flyout__item', { 'flyout__item--active': is_active })}>
-            {!should_hide_display_name && (
-                <>
-                    <div className='flyout__item-header'>
-                        <div
-                            className={classNames({
-                                'flyout__item-label--small': is_variables_get,
-                                'flyout__item-label--bold': !is_variables_get,
-                            })}
-                        >
-                            {display_name}
-                        </div>
-                        {!is_variables_get && AddButton()}
-                    </div>
-                    <div className='flyout__item-description'>
-                        {description}
-                        {onInfoClick && (
-                            <a
-                                id={display_name.replace(/\s/gi, '-')}
-                                className='flyout__item-info'
-                                onClick={onInfoClick}
+        <>
+            {is_variables_set && <Hr />}
+            <div className={classNames('flyout__item', { 'flyout__item--active': is_active })}>
+                {!should_hide_display_name && (
+                    <>
+                        <div className='flyout__item-header'>
+                            <div
+                                className={classNames({
+                                    'flyout__item-label--small': is_variables_get,
+                                    'flyout__item-label--bold': !is_variables_get,
+                                })}
```

Comment:
it's better if you use text component here but if you would like to use class you can refactor like this
```suggestion
               className = { is_variables_get ? 'flyout__item-label--small' : 'flyout__item-label--bold' }

```